content/security/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Security
+layout: default
+category: Security
+---
+
+### Security
+
+The goal of this document is to provide the most useful information for anyone interested in the security of the network. If there is missing information that would help understand and improve our network, please reach out to [contact@nycmesh.net](mailto:contact@nycmesh.net) or [join our slack][].
+
+> We are actively looking for ways to improve the security, resiliancy, and ease-of-use of the network to help the widest range of use cases. If you have ideas on how to improve anything, please [join our slack][]
+
```

Comment:
does "join our slack" link needs to be populated?

Reply:
in markdown if you provide `[my cool thing][]` it will linkify searching at the bottom of the page for `[my cool page]: https://my-cool.page`